frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.ts

```diff
@@ -113,9 +116,17 @@ export async function requestWithFallback(
     "tracingId" in layerInfo && layerInfo.tracingId != null
       ? getVolumeTracingById(state.tracing, layerInfo.tracingId)
       : null;
-  // For non-segmentation layers and for viewing datasets, we'll always use the datastore URL
-  const shouldUseDataStore = maybeVolumeTracing == null;
-  const requestUrl = shouldUseDataStore ? getDataStoreUrl() : getTracingStoreUrl();
+
+  // For non-segmentation layers and for viewing datasets, we'll always use the datastore URL.
+  // We also use the data store, if a hdf5 mapping should be locally applied. This is only the
+  // case if the proofreading tool is active or the layer was already proofread. In that case,
+  // no changes can exist on the tracing store.
+  const shouldUseDataStore =
+    maybeVolumeTracing == null || needsLocalHdf5Mapping(state, layerInfo.name);
+
```

Comment:
I think, the editable mapping is stored in the tracing store. maybe @fm3 can confirm.

to avoid confusion, I'll change "In that case, no changes can exist on the tracing store." to "In that case, no bucket data changes can exist on the tracing store.".